includes/Dashboard/Settings.php

```diff
@@ -290,15 +291,8 @@ private function remote_request( $license_key ) {
 
 		$validation['verified'] = $retrieved['data']['verified'];
 
-		// xxx handles a generic error, 4xx and 5xx handles all 400 or 500 errors.
-		$validation['error'] = 'xxx';
-		if ( 200 === $status && ! $validation['verified'] ) {
-			$validation['error'] = $retrieved['data']['errorType'];
-		} elseif ( 500 <= $status && 600 > $status ) {
-			$validation['error'] = '5xx';
-		} elseif ( 400 <= $status && 500 > $status ) {
-			$validation['error'] = '4xx';
-		}
+		$helpers             = new Helpers();
+		$validation['error'] = $helpers->validation_error( $status, $validation['verified'], $retrieved['data']['errorType'] );
```

Comment:
What is the value of this property if there are no errors? The name of this function should be changed with something more descriptive

Reply:
It will always be an "error" but two types:
- Error calling remote endpoint
- The data received from endpoint will be EXPIRED or INVALID

The messages are handled on line 233 if statement.

Should I change the function name anyway?